### DISCUSSION TITLE
Error have tried everything

### DISCUSSION THREAD

**Discussion by hatleyn** [isid0re/SoloLeveling#53] [General]
I really want to try this stript out but been trying to get it to work for hours.  I did fresh install multiple times and getting error message 
![image](https://user-images.githubusercontent.com/77400152/104474850-6dbfc980-558c-11eb-9747-d469855afcf1.png)

When i // this line i get more errors.  Please help me and inform me on what im doing wrong

**Comment by isid0re** [comment]
The hephasto error is caused by using an older version of kolbot.  Please download and install kolbot from here: 

https://github.com/blizzhackers/kolbot

After installing that version of kolbot reinstall my scripts. It should work for you then.

**Reply by hatleyn** [reply]
thanks that fixed it
